feconf.py

```diff
@@ -505,6 +506,11 @@ def get_empty_ratings():
 
 # Unfinished features.
 SHOW_TRAINABLE_UNRESOLVED_ANSWERS = False
+# Number of unresolved answers to be displayed in the dashboard for each
+# exploration.
+TOP_UNRESOLVED_ANSWERS_COUNT_DASHBOARD = 3
```

Comment:
A possibility is to do some pre-computing using e.g. continuous computations. I think that's what ultimately would need to happen but we'll need to be a bit careful to remove handled answers from that list, so maybe better to push to a future PR.

In the meantime, though, what do you think about perhaps showing the unresolved answers in the same way as we do the feedback threads -- i.e. only when a particular exploration is examined? Then we can file an issue to say: "show top answers without having to click on each exploration" or something, and that's the issue which requires pre-computation to be set up.

Also, just as a sanity check to inform this decision: let's say a dashboard has 50 explorations each with 10 states, and each state has maybe 1000 answers (where there are 200 unique answers). Can you estimate roughly how many datastore fetches are made, and how large the data transferred would be?